sphinx/application.py

```diff
@@ -29,7 +28,7 @@
     directives, roles
 
 import sphinx
-from sphinx import package_dir, locale
+from sphinx import package_dir, locale, extensions
```

Comment:
I felt that `domains`, `directives` and `writers` are plural because they contain more than one in the package. `sphinx.extensions` package aggregates features for handling extension, but it is not a collection of extensions, so I felt that singular is good. This is a trivial impression, so either is okay.